djes/mapping.py

```diff
@@ -5,20 +5,37 @@
 
 from djes.conf import settings
 
-# TODO: expand this for all django field types
 FIELD_MAPPINGS = {
     "AutoField": {"type": "long"},
-    "OneToOneField": {"type": "long"},
-    "ForeignKey": {"type": "long"},
-    "ManyToManyField": {"type": "long"},
-    "IntegerField": {"type": "long"},
+    "BigIntegerField": {"type": "long"},
+    "BinaryField": {"type": "binary"},
+    "BooleanField": {"type": "boolean"},
     "CharField": {"type": "string"},
-    "TextField": {"type": "string"},
-    "SlugField": {"type": "string", "index": "not_analyzed"},
-    "DateTimeField": {"type": "date"},
+    "CommaSeparatedIntegerField": {"type": "string"},
```

Comment:
It would be nice to map this as an `integer` and unpack the array inside ES for better search/aggregations